wagtail/api/v2/tests/test_pages.py

```diff
@@ -941,6 +941,22 @@ def test_ordering_by_unknown_field_gives_error(self):
             content, {"message": "cannot order by 'not_a_field' (unknown field)"}
         )
 
+    def test_ordering_by_id_and_slug(self):
+        response = self.get_response(order=["id,slug"])
+        content = json.loads(response.content.decode("UTF-8"))
+
+        page_id_list = self.get_page_id_list(content)
+        expected_order = [2, 4, 5, 6, 8, 9, 10, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]
+        self.assertEqual(page_id_list, expected_order)
+
+    def test_ordering_by_id_and_slug_backwards(self):
```

Comment:
This test name does not align with what you are testing, maybe copy this and make two tests;

```py
    def test_ordering_by_id_and_slug_backwards(self):
        response = self.get_response(order=["id,-slug"])
        # ...

    def test_ordering_by_id_and_slug(self):
        response = self.get_response(order=["id,slug"])
        # ...
```

Reply:
@lb-  thanks for the thorough and amazing recommendations. I will ensure to implement the necessary updates accordingly.